fortran/src/H5Fff.F90

```diff
@@ -43,6 +43,9 @@ MODULE H5F
   ! Number of objects opened in H5open_f
   INTEGER(SIZE_T) :: H5OPEN_NUM_OBJ
 
+  PRIVATE CHR_MAX
+  INTEGER, PARAMETER :: CHR_MAX=256      ! longest file name allowed on linux, 63 for func names
```

Comment:
Hmm... The lengths of the filename and pathname are defined  defined in header files, defined in bytes and depend on FS. Can we use system definition for portability?

Reply:
Good point, I will get this at build time.